other/Use-of-Unencrypted-URI-Schemes.bcheck

```diff
@@ -0,0 +1,102 @@
+metadata:
+    language: v2-beta
+    name: "Use of Unencrypted URI Schemes"
+    description: "This BCheck looks for unencrypted URI schemes ('http', 'ftp', 'ldap', 'smtp') within HTTP responses' bodies."
+    author: "Kyle Gilligan"
+    tags: "http", "ftp", "ldap", "smtp"
+
+# Commented-Out Regex:
+    # Recorded list of whitelisted variables to reduce known false positives.
```

Comment:
Do we really need these comments?

Reply:
Many thanks for your submission! It looks good. I've added a couple of really minor comments.